retrigger/triggerhandler.py

```diff
@@ -254,10 +254,10 @@ async def on_raw_message_edit(self, payload: discord.RawMessageUpdateEvent) -> N
             return
         if await self.bot.cog_disabled_in_guild(self, guild):
             return
-        if not any(t.check_edits and not t.embeds for t in self.triggers[guild.id].values()):
+        if not any(t.check_edits for t in self.triggers[guild.id].values()):
             # log.debug(f"No triggers in {guild=} have check_edits enabled")
             return
-        if "bot" in payload.data["author"]:
+        if "bot" in payload.data.get("author", []):
```

Comment:
- This is required as the URL update edit has no author field and errors on missing field.